src/Plugins/AntiW3Schools.php

```diff
@@ -0,0 +1,130 @@
+<?php declare(strict_types=1);
+
+namespace Room11\Jeeves\Plugins;
+
+use Amp\Promise;
+use Amp\Success;
+use Room11\Jeeves\Chat\Client\ChatClient;
+use Room11\Jeeves\Chat\Message\Message;
+
+class AntiW3Schools extends BasePlugin
+{
+
+    const BAD_HOST_PATTERN = 'w3schools\.com';
+    const W3S_CATEGORY_RESPONSES = [
+        'html' => '[Check the Mozilla Developer Network HTML documentation](https://developer.mozilla.org/docs/Web/HTML) for help with HTML.',
+        'css' => '[Check the Mozilla Developer Network CSS documentation](https://developer.mozilla.org/docs/Web/CSS) for help with CSS.',
+        'js' => '[Check the Mozilla Developer Network JS documentation](https://developer.mozilla.org/docs/Web/JavaScript) for help with JavaScript.',
+        'sql' => '[Check MySQL\'s official MySQL documentation](http://dev.mysql.com/doc/) for more information.',
```

Comment:
https://dev.mysql.com/doc/

Also note that W3S "sql" chapter is not specific to MySQL, also covers other horrifyingly shit database engines as well \o/